### PR TITLE
[Feature] rc-68 allow to create a schema check

### PR DESCRIPTION
**Pull request by kentwelcome**
**PR checklist**

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed
- [x] Resolve merge conflicts

**What type of PR is this?**
Feature

**What this PR does / why we need it**:
- Implement the schema check in backend
- Implement the schema check in CheckDetail View

**Which issue(s) this PR fixes**:
RC-68

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

File: recce/apis/check_api.py [hunk 1]
```diff
@@ -38,15 +39,50 @@ def create_check_from_run(name, description, run_id):
...
-    if check.run_id is None:
-        raise HTTPException(501, "Not Implemented")
-    else:
+    if check.run_id:
+        # Create check from run
         check_record = create_check_from_run(check.name, check.description, check.run_id)
         if not check_record:
             raise HTTPException(status_code=404, detail=f"Run ID '{check.run_id}' not found")
         checks_db.append(check_record)
+    elif check.model_id:
```
Comment (popcornylu) [inline recce/apis/check_api.py]: Model id is schema-check specific param, please put in `check.params.check_id`

File: recce/apis/check_api.py [hunk 2]
```diff
@@ -15,6 +15,7 @@ class CreateCheckIn(BaseModel):
     name: Optional[str] = None
     description: str = ''
     run_id: Optional[str] = None
+    model_id: Optional[str] = None
```
Comment (popcornylu) [inline recce/apis/check_api.py]: please add 

```
    type: RunType
    params: Optional[dict] = None
```
instead of model_id

**Review by popcornylu — CHANGES_REQUESTED** [review]

File: recce/apis/check_api.py (hunk 1 above)
Comment (kentwelcome) [reply recce/apis/check_api.py]: change to use `node_id`

File: recce/apis/check_api.py (hunk 2 above)
Comment (kentwelcome) [reply recce/apis/check_api.py]: change to use `node_id` and `type` has become a required parameter

**Pull request merged**